channels/slack/multiple_post/index.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright IBM Corp. 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const openwhisk = require('openwhisk');
+
+/**
+ * Examines the reply from Conversation and posts one or more replies to the channel as needed.
+ *
+ * @param  {JSON} params - output JSON to be posted back to the channel
+ * @return {JSON}        - result of the call(s) to the Post action
+ */
+function main(params) {
+  return new Promise((resolve, reject) => {
+    postMultipleMessages(params).then(result => {
+      if (result.postResponses.failedPosts.length > 0) {
+        reject(result);
+      } else {
+        resolve(result);
+      }
+    });
+  });
+}
+
+/**
+ * Checks the length of the normalized message array and sends one
+ * reply to the channel for each element in the array. If there is no message array,
+ * it just posts a single message.
+ *
+ * @param  {JSON} params - output JSON to be posted back to the channel
+ * @return {JSON}        - result of the call(s) to the Post action
+ */
+function postMultipleMessages(params) {
+  // Determine the full path of the channel's postsequence sequence
+  const packageName = extractCurrentPackageName(process.env.__OW_ACTION_NAME);
+  const deployName = packageName.split('_')[0];
+  const sequenceName = `${deployName}_postsequence`;
```

Comment:
Might be a good idea to make the sequence name a parameter of the channel package or multiple_post action. That way it's easily customizable.
It's more of a fancy feature so we can leave it for another time. For now, what you have is fine.